src/DFG.cpp

```diff
@@ -219,7 +372,7 @@ void DFG::combineForIter(list<string>* t_targetPattern){
 }
 
 // combineForUnroll is used to reconstruct "phi-add-add-..." alike patterns with a limited length.
-void DFG::combineForUnroll(list<string>* t_targetPattern){
+void DFG::combineForUnroll(list<string>* t_targetPattern, int type){
```

Comment:
Plz add comment for `type`. Is it really used inside this func?

Reply:
`type` is used in my new functions that implements fine-grained fusion and to be compatible with the previous codes, I added a new parameter `type` and set its default value to "", which not influenece the execution of previous codes.